pkgs/top-level/all-packages.nix

```diff
@@ -12068,8 +12070,17 @@ in
 
   altcoins = recurseIntoAttrs ( callPackage ../applications/altcoins {
     callPackage = newScope { boost = boost155; };
-  } );
+  } ) // (
+    let callPackage = newScope { boost = boost162; openssl = openssl_1_1_0; };
```

Comment:
let me know if [this looks good now](https://github.com/NixOS/nixpkgs/pull/19944/commits/e05082861a05700ef98eb2486d4714dea54d7211)